docs/installation/getting-started/predefined-builds.md

```diff
@@ -877,7 +877,17 @@ The table below presents the list of all plugins included in various builds. <!-
 				<td style="text-align:center; width:70px">✅</td>
 			</tr>
 			<tr>
-				<td><a href="https://ckeditor.com/docs/ckeditor5/latest/api/essentials.html">Essentials</a> *</td>
+				<td><a href="https://ckeditor.com/docs/ckeditor5/latest/api/essentials.html">Essentials</a> *<br />
+				Includes:</br />
+				<ul>
+				<li><a href="https://ckeditor.com/docs/ckeditor5/latest/api/clipboard.html">Clipboard</a></li>
+				<li><a href="https://ckeditor.com/docs/ckeditor5/latest/api/enter.html">Enter</a></li>
+				<li><a href="https://ckeditor.com/docs/ckeditor5/latest/api/select-all.html">SelectAll</a></li>
+				<li><a href="https://ckeditor.com/docs/ckeditor5/latest/api/module_enter_shiftenter-ShiftEnter.html">ShiftEnter</a></li>
```

Comment:
This link point to the plugin class documentation. The other links, point to the documentations of modules containing (among other things) the plugins. This is inconsistent. I see two possible approaches:
1. Make all the other links point to the plugin classes (docs of those, ofc.), or
2. Under the link pointing to `enter.html` mention that it contains two plugins: `Enter` and `ShiftEnter`.

Reply:
In commit.